### ISSUE TITLE
Secondary VyOS instance

### ISSUE THREAD

**Issue by JacksonWrath** [JacksonWrath/homelab-base#19]
https://docs.vyos.io/en/equuleus/configuration/highavailability/index.html

Blocked by 
- #13

**Comment by JacksonWrath** [comment]
Done. Almost exactly followed the VRRP setup from this scenario doc (just without VLANs, since that's done by the host):
https://docs.vyos.io/en/equuleus/configexamples/ha.html

Was able to configure the primary with a new IP and set the current IP as the virtual one in one shot. Then setup secondary with lower priority.


Powered off the primary instance, and it failed over basically instantly. Glorious.
